tool/settings.re

```diff
@@ -77,6 +77,23 @@ fn create_settings_menu() -> Ui {
                 }
             },
         }),
+        UiElement::FloatInput(FloatInput {
+            label: Text { text: "Log Message Duration (s)" },
+            input: f"{SETTINGS.log_message_duration.to_float() / 1000.}",
+            onclick: fn(input: string) {},
+            onchange: fn(input: string) {
+                match input.parse_float() {
+                    Result::Ok(val) => {
+                        if 0.0 <= val {
+                            let milliseconds = val * 1000.;
+                            SETTINGS.log_message_duration = milliseconds.round(3).to_int();
```

Comment:
`float::to_int` truncates all decimal places. If you have `(1.9).to_int()`, it'll return `1`. Thus you first want to round to 0 decimal places: `(1.9).round(0).to_int() == 2`. Rounding to 1 or 2 or 3 or more places doesn't have any value, as all decimals will be truncated anyways.